### DISCUSSION TITLE
질문

### DISCUSSION THREAD

**Discussion by jun5640** [naverz/zepeto-studio-kor#110] [5. Q&A]
안녕하세요?
몇 가지 궁금한 부분이 있어 글을 남깁니다.

제페토에 기존에 올라와 있는 맵들과 현재 제가 Unity로 만들고 있는 월드를 살펴보면서 궁금한 사항이 있습니다.
1. 제스쳐와 관련하여 지원해주시는 예제 혹은 코드가 있나요? 없다면 Unity에서 구현을 해야 하는 부분인가요?
2. 인벤토리 및 상점, 현재 유저의 팔로워 및 팔로잉을 지원해주는 예제 혹은 코드가 있나요?
3. 사진찍기와 관련된 부분에 대해 예제 혹은 코드가 있나요?

만약 해당 부분들에 대하여 지원을 해주시지 않는다면 추후 업데이트를 통해서 지원이 가능한 부분 인가요?

**Comment by crash-k** [comment]
안녕하세요.
기존 내부 제작 월드의 기능을 제공할 계획은 아직 없습니다.
질문주신 1, 2, 3 부분에 대해선 아직 지원 계획이 없습니다.
감사합니다.
